### DISCUSSION TITLE
Support Sqlite schema in inferred schema

### DISCUSSION THREAD

**Discussion by Sytten** [diesel-rs/diesel#3168] [Ideas]
There is already support for schemas in the schema file itself and it is supported for other databases than Sqlite.
It doesn't seem to be too hard to support it for sqlite either and I am willing to do the work for it since I happen to need it.
Currently it is just blocked here: https://github.com/diesel-rs/diesel/blob/75b8ef3b1ed6817f12b4b9504e45cedbbb5a4fd2/diesel_cli/src/infer_schema_internals/sqlite.rs#L47-L51
In theory you can execute the command multiple times for each DB, but the problem is that the schema is then not appended to the name of the table so it is annoying if at runtime you use `ATTACH DATABASE`.

Let me know if I am missing something obvious here!

**Comment by weiznich** [comment]
As already answered in the gitter chat: I'm happy to accept a PR implementing this.
